karyon3-api/src/main/java/com/netflix/karyon/annotations/Priority.java

```diff
@@ -0,0 +1,21 @@
+package com.netflix.karyon.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Use Priority with classes that are auto-loaded (such as AutoBinder) to control
+ * the order in which these auto-loaded classes are called.  
```

Comment:
Good question.  It's random now but perhaps we should throw an error instead.  I agree that a DAG would be ideal here but it would be difficult to implement since Priority is processed outside of DI (for auto binding).  Let's revisit this if/when this becomes an issue.